src/utils/changes.ts

```diff
@@ -4,6 +4,9 @@ import { getVersion } from './version';
  * Path to the changelog file in the target repository
  */
 export const DEFAULT_CHANGELOG_PATH = 'CHANGELOG.md';
+export const DEFAULT_UNRELEASED_TITLE = 'Unreleased';
+export const DEFAULT_CHANGESET_BODY = '- No documented changes.';
```

Comment:
At a first glance, I was thinking: Can these two be private?

Since you're planning to make those configurable later, however, would it make sense to move them into `prepare.ts` and have the defaulting there? Prepare has access to the configuration, and so most likely that logic will have to move anyway.

Reply:
Made them private for now. Will do the actual moving when we implement the configuration options.